ArjunaCore/arjuna/classes/com/arjuna/ats/internal/arjuna/abstractrecords/LastResourceRecord.java

```diff
@@ -38,6 +38,7 @@ public class LastResourceRecord extends AbstractRecord
 
     public LastResourceRecord(OnePhaseResource opr)
     {
+        // this means that all last resources get the same uid even if multiple last resources is enabled
```

Comment:
Remark: This means that by default, enlisting a second last resource will be rejected as a duplicate unless `CoreEnvironmentBean.allowMultipleLastResources` is enabled which is fine because that is the behaviour we want.